web_widget_many2many_tags_multi_selection/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Use short headers, as in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py (applicable to all files)